index.html

```diff
@@ -33,6 +34,6 @@
                 <a href="https://wwww.github.com/didierganthier"><img src="images/github.svg" alt="github"></a>
             </div>
```

Comment:
- [ ] [OPTIONAL] This social media icons should be structured using `ul` tag surrounding `li` tags, and each `li` tags should have an `a` tag within it. See the structure below:

```
<ul>
    <li><a href=""><img src="" alt=""></a></li>
    <li><a href=""><img src="" alt=""></a></li>
    <li><a href=""><img src="" alt=""></a></li>
    <li><a href=""><img src="" alt=""></a></li>
    <li><a href=""><img src="" alt=""></a></li>
  </ul>
```